src/components/appSwitch/AppSwitch.js

```diff
@@ -138,8 +138,9 @@ const AppSwitch = (props: LoadingProps) => {
     store.set('regMethod')(regMethod)
 
     if (isLoggedInCitizen) {
-      API.verifyTopWallet().catch(exception => {
-        const message = getErrorMessage(exception)
+      API.verifyTopWallet().catch(e => {
+        const message = getErrorMessage(e)
```

Comment:
why are we re-creating an error and not simply using the catched error?

Reply:
We have interceptor in Api.js which (in mostly cases) throws exception.response.data (e.g json body of the  request failed) instead of the source exception. Don't know why it was needed but we decided do not refactor this (by now) but just re create error objects using data.error (sent by good server ) as the message